src/components/EditorSettings.vue

```diff
@@ -21,10 +21,6 @@
 
 <template>
 	<div class="section">
```

Comment:
```suggestion
	<div>
```

This leads to the additional whitespace around the radio buttons, no? :)